From: Chris Down <chris@chrisdown.name>
Date: Mon, 30 Sep 2019 14:25:14 +0100
Subject: cgroup: docs: memory.high doc fixups

The docs just tautologically call this the "high limit". Just call it
throttling as we do in cgroup-v2.rst.

(cherry picked from commit ba79e19cb299623c9c0dd4c6a1ae80f795409b04)
(cherry picked from commit de1d25a506db17119065447f8d0225d9ae00fafc)
---
 man/systemd.resource-control.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd.resource-control.xml b/man/systemd.resource-control.xml
index b48704a..98d711b 100644
--- a/man/systemd.resource-control.xml
+++ b/man/systemd.resource-control.xml
@@ -284,7 +284,7 @@
         <term><varname>MemoryHigh=<replaceable>bytes</replaceable></varname></term>
 
         <listitem>
-          <para>Specify the high limit on memory usage of the executed processes in this unit. Memory usage may go
+          <para>Specify the throttling limit on memory usage of the executed processes in this unit. Memory usage may go
           above the limit if unavoidable, but the processes are heavily slowed down and memory is taken away
           aggressively in such cases. This is the main mechanism to control memory usage of a unit.</para>
 
@@ -292,7 +292,7 @@
           parsed as Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively. Alternatively, a
           percentage value may be specified, which is taken relative to the installed physical memory on the
           system. If assigned the
-          special value <literal>infinity</literal>, no memory limit is applied. This controls the
+          special value <literal>infinity</literal>, no memory throttling is applied. This controls the
           <literal>memory.high</literal> control group attribute. For details about this control group attribute, see
           <ulink url="https://www.kernel.org/doc/Documentation/cgroup-v2.txt">cgroup-v2.txt</ulink>.</para>
 
